### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by teckel12** [sagalbot/vue-sortable#14]
npm install vue-sortable doesn't work, where (and why) would one do this?

And what is this?  Where should this be setup?

import Vue from 'vue'
import Sortable from 'vue-sortable'

Vue.use(Sortable)

Shouldn't you simply be supplying the vue-sortable.js file, we add a script src="" command to our footer file, and then you provide a working example? (which I also can't find).

It seems you're making this 10x more complicated that it needs to be.  I can't get a "hello world" program running with what you're providing.

**Comment by sagalbot** [comment]
Hi @teckel12,

`npm install vue-sortable` most definitely does work. Have you used [npm](https://www.npmjs.com/) before? Most developers building out applications with Vue will be using some sort of build process, like `browserify` or `webpack`. In this case, `npm` is the go-to package manager. If you're building something that has multiple dependencies, using npm is a great way to manage them.

If you're not installing with npm, I recommend using npmcdn, which serves as a cdn for npm packages. You can get the main script file at `https://unpkg.com/vue-sortable`. Note that you'll also need to include SortableJS before including vue-sortable. This is the sort of dependency management that npm handles for you.

Here's a [jsbin with a complete example](http://jsbin.com/vasogus/edit?html,js,console,output).

On another note - Just because you don't understand something, doesn't mean it's more complicated than it should be. I will add how to install without npm to the readme, but for anyone familiar with it (which is what the library is designed for) install is a breeze.

**Comment by teckel12** [comment]
I'd simply like to include vue-sortable in a `<script>` tag, like I would do for any JS package.  Even Vue gives instructions on simply downloading and including with a script tag (and gives those instructions before any package manager) as most people don't need to bother with that for most applications.

The problem with using a package manager is that they don't actually put things where they should be.  I can do `npm install vue-sortable` all day and it won't add the `<script>` tags correctly in the right file in the right place.  I get using npm for a more complicated package.  But a JS package that only required Vue.js and one other JS file?  It seems this is what you want to tell people to add to use vue-sortable:

```
<script src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.0.3/vue.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/Sortable/1.4.2/Sortable.min.js"></script>
<script src="https://unpkg.com/vue-sortable@0.1.3"></script>
```

But, it appears that either vue-sortable or sortable.js isn't compatible with vue 2.0, which of course I'm using as it's the latest release.

I get that NPM attempts to make thing easier.  But it really only does for very large-scale applications.  For most projects that simply need jQuery and a few other JS tools, it gets in the way of development.  

Providing information on the dependencies (and versions) as well as a simple instructions on how to get it working would be ideal.  The NPM noise is just that.

**Comment by sagalbot** [comment]
I agree that the instructions for using browser global versions of the script should be included, they're in the readme now - that's a good call.

But we differ on what we think _most_ people do. This package gets ~1,650 installs a month from npm (I wouldn't call that noise), and I've never had a request for script tag instructions until now. That's open to interpretation, but it definitely seems the majority of people are using NPM, and using a build system in their projects.

**Comment by teckel12** [comment]
I went with Dragula and rolled my own integration.  I think a lot of the problem has to do with the huge difference between Vue v1 to v2 and most examples and JS add-ons only supporting v1.

The complexity of libraries only using package installers makes it even more challenging.  I get that some will use Node.js on the server-side.  But the VAST majority of people needing a JS library are doing it on the client side only.  I'd say well over 99% do it client side.  And besides, Node is going out of favor now by everything I read.

I also tried to install it with NPM, so I'm one of those numbers.  Problem is, it didn't do anything for me to use it client side.  I don't really need the library installed, I need the file or link to it.

I guess it's just one of those things I don't see the need for, like repositories.
